src/components/modals/confirmationDialog.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <baseModal :show="show">
+    <slot :emit-result="emitResult">
```

Comment:
I don't understand why there's a here.

Reply:
I removed it. iIt's not necessary you are right.